crates/bench/src/spacetime_module.rs

```diff
@@ -71,7 +73,14 @@ impl BenchDatabase for SpacetimeModule {
             },
             storage: if in_memory { Storage::Memory } else { Storage::Disk },
         };
-        let module = runtime.block_on(async { BENCHMARKS_MODULE.load_module(config).await });
+
+        let module = runtime.block_on(async {
+            // We keep a saved database at "crates/bench/.spacetime".
+            // This is mainly used for caching wasmtime native artifacts.
+            BENCHMARKS_MODULE
+                .load_module(config, Some(Path::new(env!("CARGO_MANIFEST_DIR"))))
```

Comment:
I'd probably want an opinion from @coolreader18 or someone else as to whether this path is logically part of the `config` here or not?

Reply:
Possibly it is, this infrastructure is only used in a few tests and benchmarks though.

Reply:
Oh, well it seems like it's just something in benchmarks, not something used by spacetimedb-core. So keep it out of Config I'd say

Reply:
Could be reevaulated later but for now it's fine